src/main/target/TINYFISH/target.h

```diff
@@ -66,7 +66,7 @@
 #define UART3_TX_PIN            PB10
 #define UART3_RX_PIN            PB11
 
-#define SBUS_TELEMETRY_UART     SERIAL_PORT_USART2
+#define SBUS_AND_TELEMETRY_UART SERIAL_PORT_USART2
```

Comment:
I didn't mean we had to remove `SERIALRX_UART`. But since `SBUS_AND_TELEMETRY_UART` is only used in `target.h` and `config.c`, it can as well be eliminated by setting the value directly in `config.c`.